cipher/rot13.c

```diff
@@ -0,0 +1,49 @@
+/*
+	Author: Luca Palumbo
+	Date: 	9.10.2019
+*/
+
+#include <stdio.h>
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+//Function usage that print how to use the program.
+void usage(char* name){
+	printf("\nUsage: %s <text>\n", name);
+	printf("Use this command to both encrypt and decrypt\n\n");
+}
+
+//Defining function rot13. It both encrypt and decrypt.
+void rot13(char* text);
+
+int main(int argc, char *argv[]){
+	int opt;
+	if (argc != 2){
+		usage( argv[0] );
+		return -1;
+	}
+	rot13(argv[1]);
+	return 1;
```

Comment:
we return **0** to indicate _Successful_ termination, & other values to indicate some error / failure.